compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -301,6 +301,8 @@ impl<'hir> Map<'hir> {
             let owner = self.tcx.hir_owner_nodes(id.owner).as_owner()?;
             let node = owner.nodes[id.local_id].as_ref()?;
             let hir_id = HirId { owner: id.owner, local_id: node.parent };
+            // HIR indexing should have checked that this.
```

Comment:
Is there a typo in this comment? The sentence seems to cut off or has an additional word.